components/MainNavBar.vue

```diff
@@ -252,6 +252,11 @@ export default {
             { name: "Glossary", url: "/glossary", external: false },
             { name: "References", url: "/references", external: false },
             { name: "Resources", url: "/resources", external: false },
+            {
+              name: "FAQ with Prof. Julian Goodare",
```

Comment:
I'm a bit concerned that this isn't a FAQ as people would expect. It might be better to rename this section to "Interview with Prof. Julian Goodare"

Reply:
good point have changed

Reply:
Looks good. Thank you for making the change